assets/js/thumbnail-clickable.js

```diff
@@ -0,0 +1,53 @@
+function toggleFullScreenImage(patternId) {
+  var fullScreenContainer = document.getElementById(
+    `fullScreenContainer${patternId}`
+  );
+
+  if (!fullScreenContainer) {
+    createFullScreenContainer(patternId);
+    document.body.style.overflow = "hidden";
+  } else {
+    document.body.removeChild(fullScreenContainer);
+    document.body.style.overflow = "auto";
+  }
+}
+
+function createFullScreenContainer(patternId) {
+  var fullScreenContainer = document.createElement("div");
+  fullScreenContainer.id = `fullScreenImageContainer${patternId}`;
+  fullScreenContainer.style.position = "fixed";
+  fullScreenContainer.style.top = "0";
+  fullScreenContainer.style.left = "0";
+  fullScreenContainer.style.width = "100%";
+  fullScreenContainer.style.height = "100%";
+  fullScreenContainer.style.backgroundColor = "rgba(0, 0, 0, 0.8)";
+  fullScreenContainer.style.zIndex = "9999";
+  fullScreenContainer.style.backdropFilter = "blur(10px)";
+  fullScreenContainer.style.display = "flex";
+  fullScreenContainer.style.alignItems = "center";
+  fullScreenContainer.style.justifyContent = "center";
```

Comment:
We can reduce code here in JS by making new CSS class and assigning it to this div, however, let's move forward here, LGTM